packages/js/client/src/__tests__/core/plugin-subinvoke.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Client, msgpackEncode, PluginModule } from "@polywrap/core-js";
+import { PolywrapClient } from "../..";
+
+describe("plugin-subinvoke", () => {
+  const mockPlugin = () => {
+    class MockPlugin extends PluginModule<Record<string, unknown>> {
+      public async call(
+        input: { input: Uint8Array },
+        client: Client
+      ): Promise<ArrayBuffer> {
+        const res = await client.invoke({
+          uri: "ens/http.polywrap.eth",
```

Comment:
Since this problem is not specific to HTTP, can we use another plugin to test it? I don't think it's ideal if our tests are making HTTP calls